### DISCUSSION TITLE
how to edit etcd DB SIZE over 2G

### DISCUSSION THREAD

**Discussion by chenxb0715** [k3s-io/k3s#6685] [Q&A] [answered]
**Environmental Info:**
K3s Version: 

Node(s) CPU architecture, OS, and Version: 

Cluster Configuration:

**Describe the bug:**

**Steps To Reproduce:**

- Installed K3s: 

**Expected behavior:**

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment, marked as answer]
Answered at https://github.com/k3s-io/k3s/issues/4787#issuecomment-1039493992
